.dockerignore

```diff
@@ -0,0 +1,45 @@
+# This file excludes paths from the Docker build context.
```

Comment:
All the new files except `docker-compose.yml` were generated with `mix phx.gen.release --docker`.